qa/L0_backend_python/test.sh

```diff
@@ -103,6 +103,29 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+# Triton non-graceful exit
+
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+# Trigger non-graceful termination of Triton
+kill -9 $SERVER_PID
+
+# Wait 10 seconds so that Python gRPC server can detect non-graceful exit
```

Comment:
Wouldn't this be dependent on `--exit-timeout-secs`? The default is 30 seconds I believe.

Reply:
It is different from that. I killed the Triton with `-9` so it will exit immediately. This is dependent on the lines below of the python_backend. It will be 6 seconds maximum, setting it to 10 to give it some margin.

https://github.com/triton-inference-server/python_backend/commit/57fa7abf497bef199071f9665cb6a879139a1a4a#diff-360c0054757ac9c0b4a774839a0b76f3R300-R333